.ci/pytorch/windows/internal/xpu_install.bat

```diff
@@ -101,6 +104,14 @@ goto xpu_install_end
 
 :xpu_bundle_install
 
+:: Install Level Zero SDK
+set XPU_EXTRA_LZ_URL=https://github.com/oneapi-src/level-zero/releases/download/v1.14.0/level-zero-sdk_1.14.0.zip
+curl -k -L %XPU_EXTRA_LZ_URL% --output "%SRC_DIR%\temp_build\level_zero_sdk.zip"
+echo "Installing level zero SDK..."
+7z x "%SRC_DIR%\temp_build\level_zero_sdk.zip" -o"%SRC_DIR%\temp_build\level_zero"
```

Comment:
It assumes `7z` has been installed. Is that always true? I will let @chuanqi129 to make the decision.

Reply:
@EikanWang Yes, we installed here: https://github.com/pytorch/pytorch/blob/main/.ci/pytorch/windows/internal/7z_install.bat